client/app/core/constants.js

```diff
@@ -1,10 +0,0 @@
-/* global _:false, ActionCable:false, $:false, sprintf: false, moment: false */
```

Comment:
Since there wasn't really anything to export from this file (everything in it is global already), I just moved the constant registrations to the `app.core` module itself. That way it is very easy to see exactly what belongs to each module.